### PR TITLE
Feature/OMEGA-220 Add metering point description to metering points table

### PR DESCRIPTION
**Pull request by ragnhildlarsen**
OMEGA-220

## What is the reason for this change?

Metering points table should have a column for metering point description according to the [Figma sketches](https://www.figma.com/file/sJHoScmQRq0hlr6AcHI9WB/Elvia.no?node-id=9560%3A55962 - Connect to preview)

### What is the expected result?

Metering point description should be shown if the metering point description has a value, otherwise it should show `-`

Screenshot 2020-09-28 at 11 06 50

## How is it solved?

Solved by adding a new term in Contentful for "Målerbeskrivelse" that was used by a new `<th>` tag. In the MeteringPointRow a new `<td>` tag was added that accessed the metering point description value. 

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |        |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?

- [ ] Metering point description should be shown in the table
- [ ] If the metering point description value is null, `-` should be shown

### CODE REVIEW

**Comment by ragnhildlarsen** [comment]
Had to merge PR for OMEGA-146 to include some changes related to using terms in the table.

**Comment by eTallang** [comment]
You probably want to convert this to a draft, since it depends on the filter task being merged :slightly_smiling_face:

**Review by t0ff3r — APPROVED** [review]

**Comment by t0ff3r** [comment]
Does the description exist for any meters yet? Or is this column empty for everyone?

File: packages/web/src/services/customer-portal-api.ts [hunk 1]
```diff
@@ -105,9 +105,9 @@ export interface MeteringPoint {
   meterSerialNumber: string
   meteringPointTypeValue: string
   isAms: boolean
+  meteringPointDescription: string
```
Comment (eTallang) [inline packages/web/src/services/customer-portal-api.ts:108]: A description is probably optional (should be confirmed)? Should in that case be marked with `?`.

**Review by eTallang — CHANGES_REQUESTED** [review]

**Comment by ragnhildlarsen** [comment]
> Does the description exist for any meters yet? Or is this column empty for everyone?

I guess some meters may have a description but the only way to set a description is to contact customer service so probably not many meters that have it.

File: packages/web/src/services/customer-portal-api.ts (hunk 1 above)
Comment (ragnhildlarsen) [reply packages/web/src/services/customer-portal-api.ts:108]: When testing the endpoint in swagger it seems like it always returns contracts with metering points that have the field meteringPointDescription. The value is null if description is not set.

Comment (eTallang) [reply packages/web/src/services/customer-portal-api.ts:108]: Maybe we could set the type to `string | null` in that case then :slightly_smiling_face:

**Review by t0ff3r — APPROVED** [review]

**Review by eTallang — APPROVED** [review]
LGTM

**Pull request merged**